### ISSUE TITLE
Deploy CDash on Kubernetes Cluster

### ISSUE THREAD

**Issue by opadron** [spack/spack#8361]
Follow up from [this](https://github.com/spack/spack/issues/7421#issuecomment-391767740) progress update.

To support the release and PR testing process, we are working on deploying an instance of CDash  on our Kubernetes cluster.  The CDash service will receive, store, and aggregate information related to the builds of each Spack package.  The CDash web application will provide a dashboard that project maintainers will use to keep track of which packages are building and which are in need of updating.

**Comment by opadron** [comment]
Here's what we've managed so far:

 - With help from @zackgalbreath, we were able to track down and fix an issue in CDash's UI (kitware/cdash-docker#1, fixed in kitware/cdash#681).

 - I finalized and completed a Docker image for general CDash deployments ([kitware/cdash-docker](https://www.github.com/kitware/cdash-docker)).

Next, I will work on learning more about [Helm](https://helm.sh/) and putting together a Helm chart for deploying CDash onto the Kubernetes cluster.  I'm going to work with the charts I've found while looking into #8362, and using what I learn from that exercise for CDash.
